Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -603,6 +605,35 @@ private extension ProductStore {
             completion(result)
         }
     }
+
+    func generateProductDetails(siteID: Int64, scannedTexts: [String], completion: @escaping (Result<ProductDetailsFromScannedTexts, Error>) -> Void) {
+        let prompt = [
+            "Write a name and description of a product for an online store given the array of scanned text strings from a packaging photo at the end.",
+            "Return only a JSON with the name in `name` field, description in `description` field, " +
+            "and the detected language as the locale identifier in `language` field.",
+            "The output should be in valid JSON format.",
+            "Identify the language detected in the array and use the same language to write the name and description.",
+            "Make the description 50-60 words or less.",
+            "Use a 9th grade reading level.",
+            "Perform in-depth keyword research relating to the product in the same language of the product title, " +
+            "and use them in your sentences without listing them out." +
+            "\(scannedTexts)"
```

Comment:
I was skeptical because the scanned texts are not surrounded by backticks, but AI seems to work well without them 🤔

Reply:
The first sentence of the prommpt "Write a name and description of a product for an online store given the array of scanned text strings from a packaging photo at the end." specifies the array is at the end, thankfully ChatGPT understands and respects that for most cases 😅